src/export/ExportMixerPanel.h

```diff
@@ -0,0 +1,54 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  ExportMixerPanel.h
+
+  Dominic Mazzoni
+
+**********************************************************************/
+
+#pragma once
+
+#include <memory>
+
+#include "widgets/wxPanelWrapper.h"
+
+#include "MemoryX.h"
+
+class wxMemoryDC;
+
+namespace MixerOptions { class Downmix; }
+
+///\class ExportMixerPanel
+///\brief Panel that displays mixing for advanced mixing option.
+class ExportMixerPanel final : public wxPanelWrapper
+{
+public:
+   ExportMixerPanel( wxWindow *parent, wxWindowID id,
+         MixerOptions::Downmix *mixerSpec, wxArrayString trackNames,
+         const wxPoint& pos = wxDefaultPosition,
+         const wxSize& size = wxDefaultSize);
```

Comment:
Not obvious in this large commit -- this is not just copied and pasted, but also the type of the third argument changes.  Please mention that in the commit comment.

Reply:
Oh but I see now -- same type, just another alias.  So never mind.